llvm/lib/Target/RISCV/RISCVTargetTransformInfo.h

```diff
@@ -308,8 +308,10 @@ class RISCVTTIImpl : public BasicTTIImplBase<RISCVTTIImpl> {
     }
     llvm_unreachable("unknown register class");
   }
-  
-  bool shouldFavorPostInc() const;
+
+  TTI::AddressingModeKind
+    getPreferredAddressingMode(const Loop *L, ScalarEvolution *SE) const;
```

Comment:
Mark this function as override

Reply:
I am not sure if this would be correct. The is already an `override` in
https://github.com/PhilippvK/corev-llvm-project/blob/4dade68c1f6dd1df29d126d6722c11951d7ef452/llvm/include/llvm/Analysis/TargetTransformInfo.h#L2052-L2056 and for none other the other heuristics there are overrides.